bin/fk.rb

```diff
@@ -19,5 +19,5 @@
 
 cmd = cmds[action]["run"]
 
-puts "fk is running #{cmd}"
-system(cmd)
+puts "fk is running #{cmd} #{ARGV[2..-1].join(' ')}"
+exec([cmd, ARGV[2..-1].join(' ')].join(' '))
```

Comment:
Why the extra join at the end?

Reply:
the first `join()` was to join all the arguments to the `cmd` into a single string then the second join is to join the original `cmd` with the `args` string. Refactored out the args to make it more clear.